README.md

```diff
@@ -0,0 +1,39 @@
+# Open Pension Scraper
```

Comment:
Re-roll from master and this should disappear (this isn't a change from this commit).